src/ui/ui_render/views/ui_app.pug

```diff
@@ -107,7 +107,7 @@ block content
       .kibanaLoader
       .kibanaWelcomeLogoCircle
         .kibanaWelcomeLogo
-    .kibanaWelcomeText
+    .kibanaWelcomeText(data-error-message='#{i18n("UI-WELCOME_ERROR")}')
```

Comment:
__issue:__ that's why CI is failing (https://kibana-ci.elastic.co/job/elastic+kibana+pull-request+multijob-intake/4504/console):
```
12:49:04 [10:49:04] →  I18N ERROR  Error in src/ui/ui_render/views/ui_app.pug
12:49:04 Error: Empty defaultMessage in i18n() or i18n.translate() is not allowed ("UI-WELCOME_ERROR").
12:49:04 ERROR  I18N ERROR  Error in src/ui/ui_render/views/ui_app.pug
12:49:04       Error: Empty defaultMessage in i18n() or i18n.translate() is not allowed ("UI-WELCOME_ERROR").
```

I guess you wanted to write something like this instead
```pug
.kibanaWelcomeText(data-error-message="#{i18n('common.ui.welcomeError', { defaultMessage: 'Kibana did not load properly. Check the server output for more information.' })}")`
```

Our tools now enforce default message for all labels that allows developers to actually see the real label content without looking into translation files. So we got rid of `en.json` entirely and extract all default messages automatically whenever translation vendors need them.

@pavel06081991 @maryia-lapata can you please help @srl295 to test this change (i.e where one should put and register test translation file for  `common.ui.` etc.)? Essentially we need a localized version of this label that includes apostrophe.

Reply:
@srl295 I tested this
```
.kibanaWelcomeText(data-error-message='#{i18n("common.ui.welcomeErrorMessage", { defaultMessage: "Kibana did not load properly. Check the server output for more information." })}')
```
with `i18n` tool. It works, the message is extracted to `en.json` by the tool (@azasypkin `common.ui` namespace is already in the [config](https://github.com/elastic/kibana/blob/master/.i18nrc.json#L3)).

Reply:
Hey @srl295! Please let us know if you need any help with this PR. We intend to ship fix for this issue in the 6.6 release, but If you don't have time that's fine too! Someone from the i18n team will just take it over.